src/System.Net.Http/tests/FunctionalTests/ManagedHandlerTest.cs

```diff
@@ -391,5 +392,103 @@ public sealed class ManagedHandler_HttpClientHandler_ConnectionPooling_Test : Ht
                 }
             }
         }
+
+        [Fact]
```

Comment:
I've noticed that new networking tests are being added as Innerloop now. We still have many networking tests as Outerloop even for the ones that use Loopback server. Is this the new plan, i.e. to add new tests as InnerLoop unless it becomes unstable?

Reply:
Honestly this is me neglecting to add Outerloop as I just copied an earlier test in the file and modified it.  But I think what you suggest is reasonable.

Reply:
I've been adding loopback tests as innerloop lately.  But yeah, we are inconsistent here.  I'm fine making loopback tests innerloop generally.